nsqadmin/static/js/models/channel.js

```diff
@@ -12,6 +12,25 @@ var Channel = Backbone.Model.extend({
         return AppState.url('/topics/' +
             encodeURIComponent(this.get('topic')) + '/' +
             encodeURIComponent(this.get('name')));
+    },
+    initialize: function() {
+      this.on('change:clients', this.calculateHostnamePort);
+    },
+    calculateHostnamePort: function(m, v) {
```

Comment:
I'd remove these arguments as they aren't used in the body of this method and can cause confusion with the `v` variable in the `forEach` method's iterator.